benchmarks/src/main/java/org/elasticsearch/benchmark/index/mapper/FallbackSyntheticSourceBlockLoaderBenchmark.java

```diff
@@ -0,0 +1,3 @@
+package org.elasticsearch.benchmark.index.mapper;
+
+public class FallbackSyntheticSourceBlockLoaderBenchmark {}
```

Comment:
Is the intend to include the benchmark?

Reply:
I didn't intend to do that, i am not sure how this ended up being here :)